zketcd.go

```diff
@@ -196,7 +196,9 @@ func (z *zkEtcd) Delete(xid Xid, op *DeleteRequest) ZKResponse {
 		}
 
 		s.Put("/zk/cver/"+pp, "")
-		s.Put("/zk/mtime/"+pp, encodeTime())
+
+		// deleting a znode will NOT update its parent mtime
```

Comment:
Who knows, I just xchk it; the zookeeper server is the ground truth. It matches the Create behavior, though, so at least it's consistent.